qiskit/pulse/instructions/phase.py

```diff
@@ -89,3 +93,45 @@ def __call__(self, channel: PulseChannel) -> 'ShiftPhase':
                       "example, ShiftPhase(3.14)(DriveChannel(0)) should be replaced by "
                       "ShiftPhase(3.14, DriveChannel(0)).", DeprecationWarning)
         return ShiftPhase(self.phase, channel)
+
+
+class SetPhase(Instruction):
+    r"""The set phase instruction sets the phase of the proceeding pulses on that channel
+    to ``phase`` radians.
+
+    In particular, a PulseChannel creates pulses of the form
+
+    .. math::
+
+        Re[\exp(i 2\pi f jdt + \phi) d_j]
+
+    The ``SetPhase`` instruction causes :math:`\phi` to be determined by the instruction's
+    ``phase`` operand.
+    """
+
+    def __init__(self, phase: float,
```

Comment:
Could we put `phase` on a new line just to be a more asthetic?

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4540/commits/998418f459f6d15d402113a689f3922b7be11e3f.